wgpu-rs/src/lib.rs

```diff
@@ -273,14 +277,17 @@ impl Device {
             .map(|binding| wgn::Binding {
                 binding: binding.binding,
                 resource: match binding.resource {
-                    BindingResource::Buffer { ref buffer, ref range } => {
-                        wgn::BindingResource::Buffer(wgn::BufferBinding {
-                            buffer: buffer.id,
-                            offset: range.start,
-                            size: range.end,
-                        })
+                    BindingResource::Buffer {
```

Comment:
is that formatting what rustfmt produces? I think the old one is better, although having no hard feelings for it

Reply:
I think there have been significant changes to rustfmt since the version last run on this code. I've been disabling rust fmt to avoid rewriting significant portions of the file.